components/base/DaySelect.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <b-field :label="$t('offer expires in')">
+    <b-select
+      placeholder="Select day to expire"
+      v-model="selectedDays"
+      expanded>
+      <option disabled selected value="">--</option>
+      <option v-for="option in days" :value="option" :key="option">
+        {{ option }} {{ $t('days') }}
```

Comment:
pls add the translation key to `en.json` for `offer expires in` 、`Select day to expire`、`days`

Reply:
done